incubator/elasticsearch/templates/master-statefulset.yaml

```diff
@@ -116,14 +116,23 @@ spec:
       - name: config
         configMap:
           name: {{ template "elasticsearch.fullname" . }}
+{{- if not .Values.master.persistence.enabled }}
```

Comment:
nit; this can be indented

Reply:
Sure thing. Will have an update for this.